ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -208,6 +207,36 @@ Verify User Can Autoscale Using Concurrency
     [Teardown]   Clean Up Test Project    test_ns=autoscale-con
     ...    isvc_names=${model_name}
 
+Verify User Can Validate Scale To Zero
+    [Tags]    ODS-2379    WatsonX
+    [Setup]    Set Project And Runtime    namespace=autoscale-zero
+    ${flan_model_name}=    Set Variable    flan-t5-small-caikit
+    ${model_name}=    Create List    ${flan_model_name}
+    Compile Inference Service YAML    isvc_name=${flan_model_name}
+    ...    sa_name=${DEFAULT_BUCKET_SA_NAME}
+    ...    model_storage_uri=${FLAN_STORAGE_URI}
+    Deploy Model Via CLI    isvc_filepath=${LLM_RESOURCES_DIRPATH}/caikit_isvc_filled.yaml
+    ...    namespace=autoscale-zero
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${flan_model_name}
+    ...    namespace=autoscale-zero
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_model_name}   namespace=autoscale-zero
+    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
+    ${header}=    Set Variable    'mm-model-id: ${flan_model_name}'
+    Query Model With GRPCURL   host=${host}    port=443
+    ...    endpoint="caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
+    ...    json_body=${body}    json_header=${header}
+    ...    insecure=${TRUE}
+    Update Scale To Zero For Knative Serving   serving_name=knative-serving
+    ...    namespace=knative-serving   status=true
+    Scale Number Of Replicas    n_replicas=0    model_name=${flan_model_name}
```

Comment:
we probably should rename this kw to "Set Minimum Replicas Number" now

Reply:
Agreed.